assets/css/style.css

```diff
@@ -252,3 +229,91 @@ hr {
   border: 0.5px;
   margin-top: 10px;
 }
+
+.chev i {
+  color: #acb7c3;
+}
+
+.contact-me {
+  background: #fff;
+}
+
+.contact {
+  background-image: url("../images/contact-bg.svg");
+  background-position: center 100px;
+  background-size: auto;
+  background-repeat: no-repeat;
+  border-top-left-radius: 160px;
+  background-color: #6070ff;
+}
```

Comment:
Kindly ensure that the design of the contact form is as close to the design as possible. It will be good to remove the overflow to let the form and the entire page to display properly and for better user experience.

Your design:
![image](https://github.com/Izebiafe/my-protfolio/assets/20554583/f7e628ca-582c-496c-a58e-95859dd34238)

Figma design:
![image](https://github.com/Izebiafe/my-protfolio/assets/20554583/e6d7e3c7-7aef-4ced-8e6a-db71b06c3b7c)